task/buildah-remote-oci-ta/0.1/buildah-remote-oci-ta.yaml

```diff
@@ -412,7 +412,8 @@ spec:
       REMOTESSHEOF
       chmod +x scripts/script-build.sh
       rsync -ra scripts "$SSH_HOST:$BUILD_DIR"
-      ssh $SSH_ARGS "$SSH_HOST" $PORT_FORWARD podman  run $PODMAN_PORT_FORWARD \
+      ssh $SSH_ARGS "$SSH_HOST" $PORT_FORWARD podman run $PODMAN_PORT_FORWARD \
+       --tmpfs "/run/secrets" \
```

Comment:
I think this will break the ADDITIONAL_SECRET feature

https://github.com/konflux-ci/build-definitions/blob/881800cb75b75724962152ce68a8b4e38e558e63/task/buildah/0.1/buildah.yaml#L314

Reply:
Ah, nevermind - this is a mount for the podman container that runs the build, not for the build itself